app/forms/flood_risk_engine/steps/base_postcode_form.rb

```diff
@@ -0,0 +1,20 @@
+module FloodRiskEngine
+  module Steps
+    class BasePostcodeForm < BaseRedirectableForm
+
+      property :post_code, virtual: true
```

Comment:
I see @reggieb's point though. Personally I'd call it `postcode` and have done with it ;-)
`post_code` as opposed to `postcode` doesn't tell a future develop much but could lead to confusion and introduction of a bug even. As an alternative what about `tmp_postcode` which has a bit more meaning?